jsx/fatal_error.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { FaGithub, FaGlobe } from 'react-icons/fa'
+const RefreshTime = 4000
+export default class FatalError extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      up: true
+    }
+  }
+  componentDidMount () {
+    this.timer = setInterval(() => {
+      this.checkHealth()
+    }, RefreshTime)
+  }
+  componentWillUnmount () {
+    window.clearInterval(this.timer)
+  }
+  checkHealth () {
+    let that = this
```

Comment:
we don't need to use redux here. Because this call is not meant to act on the application state but only on the component.

Reply:
In this context, I feel it may be ok, but the rest of the system will be unaware of the problem.  Since it looks like there is no way to remove the modal during the problem, the rest of the system is not accessible to the user.

Reply:
Exactly @Ranthalion :)